lib/systems/platforms.nix

```diff
@@ -307,6 +340,43 @@ rec {
     uboot = null;
   };
 
+  scaleway-c1 = armv7l-hf-multiplatform // {
+    gcc = {
+      cpu = "cortex-a9";
+      fpu = "vfpv3";
+      float = "hard";
+    };
+  };
+
+  utilite = {
```

Comment:
Unrelated to your review, but do you have a NixOS config for utilite too? I have one and have been wondering if I can install NixOS on it easily